doc/api/buffer.markdown

```diff
@@ -392,6 +392,19 @@ byte from the original Buffer.
     // abc
     // !bc
 
+
+### buf.indexOf(value[, byteOffset])
+
+* `value` String, Buffer or Number
+* `byteOffset` Number, Optional, Default: 0
+* Return: Number
+
+Operates similar to
+[Array#indexOf()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf).
```

Comment:
> Unfortunately that removes the code formatting to let the user know that the text is a link.

My point is that there is _no_ code formatting on this text and it shoudl be added, or are you pointing to a deficiency in our styles that should be fixed?

`grep '\[`' doc/api/*` says that we're using this pattern a lot already and I don't really see a problem.